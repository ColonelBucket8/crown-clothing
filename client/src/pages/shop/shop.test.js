import React from "react";
import { shallow } from "enzyme";
import ShopPage from "./shop.component";

it("should render shop page", () => {
  expect(
    shallow(
      <div>
        <ShopPage />
      </div>
    )
  ).toMatchSnapshot();
});
