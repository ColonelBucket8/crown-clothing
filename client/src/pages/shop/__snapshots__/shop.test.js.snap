// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render shop page 1`] = `
<div>
  <ShopPage />
</div>
`;
